src/xpk/core/kueue.py

```diff
@@ -480,30 +479,50 @@ def install_kueue_crs(
 
 
 def get_kueue_covered_resources_config(
-    cluster_hardware_name, resource_type, total_chips
+    cluster_hardware_name, resource_type, total_chips, enable_pathways
 ) -> str:
   """Gets Kueue covered resources configuration.
 
   Args:
     cluster_hardware_name: cluster hardware name.
     resource_type: resource type of tpu or gpu.
     total_chips: total number of chips for the specific resource type.
+    enable_pathways: if pathways is enabled.
 
   Returns:
     A string of Kueue covered resources configuration.
   """
+  pathways_resources = ''
+  if enable_pathways:
+    pathways_resources = """
+    - name: cpu-user
```

Comment:
I don't think you can have two resource flavors with the same covered resources, IIUC. Please test before submitting this change.